src/commands/MigrationRevertCommand.ts

```diff
@@ -45,6 +49,7 @@ export class MigrationRevertCommand implements yargs.CommandModule {
                 subscribers: [],
                 synchronize: false,
                 migrationsRun: false,
+                migrationsForce: args["force"] ? true : connectionOptions.migrationsForce,
```

Comment:
probably its better to check if its args["force"] === "1" || args["force"] === "true"